docs/topics/spider-middleware.rst

```diff
@@ -451,3 +451,86 @@ UrlLengthMiddleware
 
       * :setting:`URLLENGTH_LIMIT` - The maximum URL length to allow for crawled URLs.
 
+
+StickyMetaParamsMiddleware
+--------------------------
+
+.. module:: scrapy.spidermiddlewares.stickymeta
+   :synopsis: Spider Middleware that forwards meta parameters through requests
+
+.. class:: StickyMetaParamsMiddleware
+
+   When enabled and configured, this middleware forwards the desired :ref:`Request.meta <topics-request-meta>`
+   parameters between ``Requests`` and ``Responses``.
+
+   The :class:`StickyMetaParamsMiddleware` can be configure though the ``sticky_meta_keys``
+   spider attribute.
+
+      * :attr:`~scrapy.spiders.Spider.sticky_meta_keys` - List of keys to be stickied.
```

Comment:
:lipstick: Instead of having a 1-item list here, I would remove this line and move the link to the paragraph above.

Reply:
Done :heavy_check_mark: